src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function getSum(firstValue) {
```

Comment:
Dont need to name a function. Check [checklist](https://github.com/mate-academy/js_sum-with-closure/blob/master/checklist.md) pt.2